examples/apps/helloworld/app.py

```diff
@@ -16,11 +16,22 @@
 import json
 from flask import Flask, request
 app = Flask(__name__)
+version = ""
 
 @app.route('/hello')
 def hello():
-    service_version = os.environ.get('A8_SERVICE').split(':')
-    version = service_version[1] if len(service_version) == 2 else 'UNVERSIONED'
+    global version
+    service_version = ""
+
+    if not version:
+        if os.environ.get('A8_SERVICE'):
+            service_version = os.environ.get('A8_SERVICE').split(':')
+            if len(service_version) == 2:
+                version = service_version[1]
+        elif os.environ.get('A8_HELLO_VERSION'):
```

Comment:
How about just SERVICE_VERSION ? The example will be generic enough to be re-used elsewhere.